gamemodes/PPC_Trucking.pwn

```diff
@@ -3,6 +3,8 @@
 
 #pragma unused ret_memcpy
 
+#pragma semicolon 1
```

Comment:
Explicit programming instead of leaving stuff in default... ?

Reply:
Fair point! Since it does not require extra configuration settings we can add it here. But please add a short description of it above in a comment. The title of the commit should work.